subprojects/core-api/src/main/java/org/gradle/api/tasks/TaskOutputFilePropertyBuilder.java

```diff
@@ -16,14 +16,11 @@
 
 package org.gradle.api.tasks;
 
-import org.gradle.internal.HasInternalProtocol;
-
 /**
  * Describes an output property of a task that contains zero or more files.
  *
  * @since 3.0
  */
-@HasInternalProtocol
```

Comment:
I created https://github.com/gradle/gradle/issues/8236 to clear up this situation.